source/getting-started/javascript.md

```diff
@@ -90,48 +95,289 @@ See also: [app authorization in the API reference](/reference/#authorizing-your-
+var options;
+
+// Here we will get all streams (including root and trashed streams)
+option = {
+    // If null, retrieve active streams only
+    state: 'all'
+};
+
+// Same as above but in a selected stream structure
+options = {
+    parentId: 'valid-stream-id',
+    state: 'all'
+};
+
+connection.streams.get(options, function (err, streamList) {
+  // ...
+});
+```
+
+#### Create
+
+```javascript
+// If no id is set, one is generated;
+// If parentId is null, a "root" stream is created
+var stream = {
+  name: 'A Stream',
+  id: 'a-stream-id',
+  parentId: 'valid-stream-id'
+};
+
+connection.streams.create(stream, function (err, streamCreated) { 
+  // ...
+});
+```
+
+#### Update
+
+```javascript
+// Here we update the name of the stream created above
+stream.name: 'An Updated Stream';
+connection.streams.update(stream, function (err, streamUpdated) {
+  // ...
+});
+```
+
+#### Delete
+
+```javascript
+connection.streams.delete(stream, function (err, streamDeleted) {
+  // ...
+}, mergeEventsWithParent);
+```
+
+### Manage accesses
+
+#### Retrieve
+
+```javascript
+connection.accesses.get(function (err, accesses) {
+  // ...
+});
+```
+
+#### Create
+
+```javascript
+var access = {
+  name: 'An Access',
+  permissions: [
+    {
+      streamId: 'valid-stream-id',
+      level: 'manage'
+    }
+  ]
 };
-connection.events.create(eventData, function (err, event) { 
+
+connection.accesses.create(access, function (err, accessCreated) { 
   // ...
 });
 ```
 
 #### Update
 
 ```javascript
-event.content = 'Updated content.';
-event.update(function (err, updatedEvent) {
+// Here we update the name and permissions of the access created above;
+access.name: 'An Updated Access';
+access.permissions[0].level: 'contribute';
+connection.accesses.update(access, function (err, accessUpdated) {
   // ...
 });
 ```
 
 #### Delete
 
 ```javascript
-event.delete(function (err, trashedEventOrNull) {
+connection.accesses.delete(access, function (err, accessDeletion) {
   // ...
 });
 ```
 
+### Batch call
+
+```javascript
+var methodsData = [
+  {
+    'method': 'streams.create',
+    'params': {
+      'id': 'a-new-stream',
+      'name': 'A New Stream'
+    }
+  },
+  {
+    'method': 'events.create',
+    'params': {
+      'streamId': 'a-new-stream',
+      'type': 'note/txt',
+      'content': 'This is a new event.'
+    }
+  },
+  {
+    'method': 'accesses.create',
+    'params': {
+      'name': 'A New Access',
+      'permissions': [
+        {
+          'streamId': 'a-new-stream',
+          'level': 'read'
+        }
+      ]
+    }
+  }
+];
+
+connection.batchCall(methodsData, function (err, results) {
+  //...
+});
+```
+
+### Monitors
+
+Monitors will watch the changes from a selected structure of data (i.e: Errors, Events, Streams or Filters), made within a user account in an app. They are used to fetch the current state of all the elements in an app after load. Therefore it allows to manage data in a user account while an app in running.
```

Comment:
*Monitors watch the changes to selected data structures (i.e: Errors, Events, Streams or Filters). They are used to fetch the current state of all the elements in an app upon loading. Therefore it allows to manage data in a user account at runtime.